### PR TITLE
Followupbatchofqcmetrics

### PR DESCRIPTION
**Pull request by mwalzer**
following #155

### CODE REVIEW

**Review by edeutsch — APPROVED** [review]

File: psi-ms.obo [hunk 1]
```diff
@@ -21880,6 +21880,26 @@ name: Lipid shorthand identification confidence - Complete structure
...
+def: "The number of protein groups that pass the threshold to be considered identified with sufficient confidence." [PSI:PI]  
+is_a: MS:1002405 ! protein group-level result list attribute 
+is_a: MS:4000003 ! single value 
+relationship: has_metric_category MS:4000008 ! ID based metric 
+relationship: has_value_type xsd:int ! The allowed value-type for this CV term 
+relationship: has_units UO:0000189 ! count unit 
+
+[Term] 
+id: MS:1003328 
+name: count of identified proteoforms
```
Comment (bittremieux) [inline psi-ms.obo:21895]: **nitpick:** Use "number" instead of "count" here to be consistent?

File: psi-ms.obo [hunk 2]
```diff
@@ -22634,25 +22654,77 @@ relationship: has_metric_category MS:4000012 ! single run based metric
...
 
-[Term]
-id: MS:4000102
-name: number of detected quantification data points
-def: "The number of datapoints detected for quantification purposes within the run. These datapoints may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the meta data or analysis methods section of the recording file for the respective run." [PSI:MS]
-is_a: MS:4000003 ! single value
+[Term] 
+id: MS:4000102 
+name: number of detected quantification data points 
+def: "The number of datapoints detected for quantification purposes within the run. These datapoints may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the meta data or analysis methods section of the recording file for the respective run." [PSI:MS]
```
Comment (bittremieux) [inline psi-ms.obo:22660]: **nitpick:** "datapoints" → "data points", "meta data" → "metadata". Here and in the following terms.

**Review by bittremieux — APPROVED** [review]
Looks good. I suggested a few spelling fixes for consistency.

File: psi-ms.obo (hunk 1 above)
Comment (mwalzer) [reply psi-ms.obo:21895]: :+1:

**Pull request merged**